### ISSUE TITLE
Export individual plots as png

### ISSUE THREAD

**Issue by paulmueller** [ZellMechanik-Dresden/ShapeOut#62]
Allow to export individual plots as a PNG file.

There seems to be a chaco-related problem on at least Linux and Max OSx that makes such a feature currently unavailable:

https://github.com/enthought/chaco/issues/295

**Comment by phidahl** [comment]
Would it be possible to export the individual plots in another format such as svg or pdf?

**Comment by paulmueller** [comment]
Yes, it would be possible to export single plots as pdf. If the file format does not play a big role, this could be a solution.

**Comment by phidahl** [comment]
Then could you add a single plot export option for pdf? svg would be even better. 

For the workflow exporting all with a number prefix at once would be best. 

Thanks!
